src/olympia/reviewers/tests/test_models.py

```diff
@@ -386,98 +386,176 @@ class TestReviewerSubscription(TestCase):
+        GroupUser.objects.create(
+            group=self.listed_reviewer_group, user=self.admin_reviewer)
+        GroupUser.objects.create(
+            group=self.unlisted_reviewer_group, user=self.admin_reviewer)
+        # Don't subscribe admin to updates yet, will be done in tests.
+
+    def test_send_notification(self):
+        subscription = ReviewerSubscription.objects.get(
+            user=self.listed_reviewer)
+        subscription.send_notification(self.listed_version)
         assert len(mail.outbox) == 1
-        assert mail.outbox[0].to == [u'del@icio.us']
+        assert mail.outbox[0].to == [u'listed@reviewer']
         assert mail.outbox[0].subject == (
-            'Mozilla Add-ons: Delicious Bookmarks Updated')
+            'Mozilla Add-ons: SubscribingTest Updated')
+
+    def test_send_notifications(self):
+        another_listed_reviewer = user_factory(email='listed2@reviewer')
+        GroupUser.objects.create(
+            group=self.listed_reviewer_group,
+            user=another_listed_reviewer)
+        ReviewerSubscription.objects.create(
+            addon=self.addon, user=another_listed_reviewer,
+            channel=amo.RELEASE_CHANNEL_LISTED)
 
-    def test_notifications(self):
-        send_notifications(sender=Version, instance=self.version)
+        send_notifications(sender=Version, instance=self.listed_version)
         assert len(mail.outbox) == 2
         emails = sorted([o.to for o in mail.outbox])
-        assert emails == [[u'del@icio.us'], [u'regular@mozilla.com']]
+        assert emails == [[u'listed2@reviewer'], [u'listed@reviewer']]
 
     def test_notifications_setting_persists(self):
-        send_notifications(Version, self.version)
+        send_notifications(Version, self.listed_version)
         assert ReviewerSubscription.objects.count() == 2
         mail.outbox = []
-        send_notifications(Version, self.version)
-        assert len(mail.outbox) == 2
+        send_notifications(Version, self.listed_version)
+        assert len(mail.outbox) == 1
+        mail.outbox = []
+        send_notifications(Version, self.unlisted_version)
+        assert ReviewerSubscription.objects.count() == 2
+        mail.outbox = []
+        send_notifications(Version, self.unlisted_version)
+        assert len(mail.outbox) == 1
 
-    def test_send_notifications_unlisted(self):
-        self.reviewer_group = Group.objects.create(
-            name='Reviewers: Unlisted', rules='Addons:ReviewUnlisted')
-        GroupUser.objects.create(
-            group=self.reviewer_group, user=self.user_one)
-        version_uploaded.send(sender=Version, instance=self.version)
-        assert len(mail.outbox) == 2
-        assert mail.outbox[0].to == [u'del@icio.us']
+    def test_listed_subscription(self):
+        version_uploaded.send(sender=Version, instance=self.listed_version)
+        assert len(mail.outbox) == 1
+        assert mail.outbox[0].to == [u'listed@reviewer']
```

Comment:
`u'`  :whatyearisthis:

Reply:
I took that from the old test. I guess it's not needed anymore?

Reply:
we've not bothered to strip it out of all the old code, but it's jarring to see it in new code.  That said, meh, I wouldn't bother with a new commit _just_ for that.

Reply:
Not a big deal, I will remove it.